js/plugins/drupalentity/plugin.js

```diff
@@ -211,54 +211,4 @@
     return 'entity-embed-' + generateEmbedId.counter++;
   }
 
-  /**
-   * Attaches or detaches behaviors, except the ones we do not want.
-   *
-   * @param {string} action
-   *   Either 'attach' or 'detach'.
-   * @param context
-   *   The context argument for Drupal.attachBehaviors()/detachBehaviors().
-   * @param settings
-   *   The settings argument for Drupal.attachBehaviors()/detachBehaviors().
-   */
-  function runEmbedBehaviors(action, context, settings) {
```

Comment:
So I had already written the AJAX embedding abstraction (sorry @cs-shadow). I added it to https://github.com/drupal-media/entity_embed/pull/161. I might just merge that PR now and file follow-up issues for what's left to do...